src/App.css

```diff
@@ -133,3 +133,93 @@ nav {
 a.active {
   text-decoration: underline;
 }
+
+/* Pages &  Page */
+
+.pages {
```

Comment:
"page" is pretty generic. any chance we can rename this to something more representative such as "list"?

Reply:
Thanks! We changed to location (`list__locations`). I hope it's much better.

![image](https://user-images.githubusercontent.com/8689897/126822667-f7ca979c-e4cf-4a0a-b88d-55a02991f572.png)